WeakAurasOptions/GenericTrigger.lua

```diff
@@ -67,6 +64,38 @@ function WeakAuras.GetGenericTriggerOptions(data, trigger, untrigger)
   end
 
   local options = {
+    typedesc = {
```

Comment:
This moved from the general trigger options to the generic trigger options, as selection between status and event is generic trigger specific